staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -122,6 +124,11 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"The maximum number of mutating requests in flight at a given time. When the server exceeds this, "+
 		"it rejects requests. Zero for no limit.")
 
+	fs.Float64Var(&s.DisconnectChanceOnMaxRequests, "disconnect-chance-on-max-requests", s.DisconnectChanceOnMaxRequests, ""+
```

Comment:
default is populated in the generated help text from the value passed into Float64Var, we don't need to explicitly mention it in the help text